extensions/twitch/src/streamer.tsx

```diff
@@ -47,13 +44,10 @@ export default function main() {
           <ActionPanel>
             <OpenAction title="Open Channel" target={`https://twitch.tv/${item.broadcaster_login}`} />
```

Comment:
This action appears almost everywhere! You can create utils.tsx and link them here